config/smart-agent-core.js

```diff
@@ -1,6 +1,11 @@
 exports['default'] = {
-  smartAgentCore: (api) => {
+  // believe cupboard trip message rebel guess moral churn clean cram journey ready
+  ethAccounts: {
+    '0x66D487419Ac030993Dba3D893Fc99bdafDCf60bB': 'cf316cf854f6d4ae7bd7c28ad124ae6ff95e3f4f7439e67a85689c0b8f632c13'
+  },
+  smartAgentCore: () => {
```

Comment:
(belongs to question group 1)
❓ Does the seed need its own runtime? Derived smart agents will have a running runtime and the seed itself will most probably never use its own `runtime` and auth. So every Serve would have an extra runtime with default accounts and a key exchange listener, that does nothing or am I wrong?
imho we could move the `_ensureAuth` into the class scope and use the `runtime` from the current instance... or? ^^
```suggestion
```